libvncclient/listen.c

```diff
@@ -30,11 +30,12 @@
 #endif
 #include <sys/types.h>
 #ifdef WIN32
-#define close closesocket
```

Comment:
Just from looking at it I suspect replacing this line with
```
#define close closeSocket
```
(note the uppercase S) should be enough to make things work without changing anything else in this file. Correct me if I'm wrong :)